website/docs/r/container_app.html.markdown

```diff
@@ -0,0 +1,337 @@
+---
+subcategory: "Container Apps"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_container_app"
+description: |-
+  Manages a Container App.
+---
+
+# azurerm_container_app
+
+Manages a Container App.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "acctest-01"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+  retention_in_days   = 30
+}
+
+resource "azurerm_container_app_environment" "example" {
+  name                       = "myEnvironment"
+  location                   = azurerm_resource_group.example.location
+  resource_group_name        = azurerm_resource_group.example.name
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.example.id
+}
+resource "azurerm_container_app" "example" {
+  name                         = "example"
+  container_app_environment_id = azurerm_container_app_environment.example.id
+  resource_group_name          = azurerm_resource_group.example.name
+  revision_mode                = "Single"
+
+  template {
+    container {
+      name   = "acctest-cont-%[2]d"
+      image  = "mcr.microsoft.com/azuredocs/containerapps-helloworld:latest"
+      cpu    = 0.25
+      memory = "0.5Gi"
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `container_app_environment_id` - (Required) The ID of the Container App Environment to host this Container App. Changing this forces a new resource to be created.
+
+* `name` - (Required) The name for this Container App. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which the Container App Environment is to be created. Changing this forces a new resource to be created.
+
+* `revision_mode` - (Required) The revisions operational mode for the Container App. Possible values include `Single` and `Multiple`.
+
+* `template` - (Required) A `template` block as detailed below.
+
+---
+
+* `dapr` - (Optional) A `dapr` block as detailed below.
```

Comment:
whats dapr? and should we un-abbrivate it?

Reply:
CTX: https://dapr.io/
It's the "product name" of the system, and how it's typically referred to, so again feels appropriate to use the shorthand here.